CMSgov/mint-app
# [EASI-2589] Fix: Migration Naming Conflict

# EASI-2589

## Changes and Description

- Fixed merge issue where two migrations prefixed with 43__x



## How to test this change

- Create a model plan
- Update the status of that model plan to 'Paused' and 'Canceled'
- Confirm that the corresponding database entry updates according to the model plan status changes

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
